app/models/project_media.rb

```diff
@@ -553,6 +553,23 @@ def add_nested_objects(ms)
     ms.attributes[:requests] = requests
   end
 
+  def get_similar_articles
+    team = project_media.team
+    # Get search query based on Media type
+    # Quote for Claim
+    # Transcription for UploadedVideo , UploadedAudio and UploadedImage
+    # Title and/or description for Link
+    media = self.media
+    search_query = case media.types
```

Comment:
This should be `type`, not `types`. If this was not raised by a unit test, please add one.

Reply:
Did not add a unit test for this one but also I see no tests run for the PR is this normal?

Reply:
I think it's because the target branch is not `develop`? But it should run. @chinelo-obitube, can you please check?